website/docs/networking/vpc-lattice/cleanup.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Cleanup"
```

Comment:
I don't see any changes to reset-environment. User should be able to run the reset-environment script at any point of this workshop to undo these changes. This is required before we can merge.